server/views/pages/licence/partials/signatures.njk

```diff
@@ -0,0 +1,15 @@
+{% macro signatures(releaseDate, approverName, approvedDate, offenderName) %}
+    <div class="boxed">
+        <div class="signatures">
+            <p>Signed:  {{ approverName }}  for the Secretary of State for Justice.</p>
+            <p>Date: {{ approvedDate | datetimeToDate }}</p>
+            <p></p>
+            <p>This licence has been give to me and its requirements have been explained.</p>
+            <p></p>
+            <p>Name: <span class="bold">{{ offenderName }}</span></p>
+            <p>Signed:</p>
+            <textarea class="signature-box" rows="4" cols="60" readonly draggable="false"></textarea>
```

Comment:
Feels like `textarea` is probably not what you want to use here, and might even throw accessability errors on sortsite since its outside of a `<form>` tag? Maybe just a styled div?

Reply:
Yes, it was a bit dirty. I will address this as a styled div.